pkg/appconsts/consensus_consts.go

```diff
@@ -4,5 +4,5 @@ import "time"
 
 const (
 	TimeoutPropose = time.Second * 10
-	TimeoutCommit  = time.Second * 15
+	TimeoutCommit  = time.Second * 11
```

Comment:
for BSR, we had 10 and 10, which typically resulted in 12-13 second empty blocks. This should allow for 13-14 second empty blocks. If there is consistent load on the network, then we should hit 13-15 seconds.

Reply:
the empty blocks just means that we won't need the extra 10 seconds to gossip proposals and prevote. With large bocks, tednermint's gossiping stack is rather inefficient atm, so we need the extra time to consistently gossip large blocks. When that occurs, we end up with up to 22 second block times. This was one of the goals with having a dynamic value for the 11 seconds set above. Instead of consistently waiting 11 seconds, we would change how long we wait depending on how long we have already waited.